dev-libs/argtable/argtable2-13.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="An ANSI C command line parser"
+DESCRIPTION=" Argtable is an ANSI C library for parsing GNU style command \
+line options with a minimum of fuss. It enables a program's command line \
+syntax to be defined in the source code as an array of argtable structs. \
+The command line is then parsed according to that specification and the \
+resulting values are returned in those same structs where they are accessible \
+to the main program."
+HOMEPAGE="http://argtable.sourceforge.net/"
+COPYRIGHT="1998-2011 Stewart Heitmann"
```

Comment:
```suggestion
COPYRIGHT="1998-2001, 2003-2011 Stewart Heitmann"
```

I don't think we should collapse the year like this, given that the author have explicitly put it into two different blocks.